docs/framework/network-programming/tls.md

```diff
@@ -15,70 +15,143 @@ helpviewer_keywords:
 
 The following sections show how to verify you're not using a specific TLS or SSL version.
 
-### For HTTP networking
+#### For HttpClient and HttpWebRequest
 
 <xref:System.Net.ServicePointManager>, using .NET Framework 4.7 and later versions, will use the default security protocol configured in the OS. To get the default OS choice, if possible, don't set a value for the <xref:System.Net.ServicePointManager.SecurityProtocol?displayProperty=nameWithType> property, which defaults to <xref:System.Net.SecurityProtocolType.SystemDefault?displayProperty=nameWithType>.
 
-Because the <xref:System.Net.SecurityProtocolType.SystemDefault?displayProperty=nameWithType> setting causes the <xref:System.Net.ServicePointManager> to use the default security protocol configured by the operating system, your application may run differently based on the OS it's run on. For example, Windows 7 SP1 uses TLS 1.0 while Windows 8 and Windows 10 use TLS 1.2.
-
-The remainder of this article is not relevant when targeting .NET Framework 4.7 or later versions for HTTP networking.
+Because the <xref:System.Net.SecurityProtocolType.SystemDefault?displayProperty=nameWithType> setting causes the <xref:System.Net.ServicePointManager> to use the default security protocol configured by the operating system, your application may run differently based on the OS it's run on. For example, Windows 10 uses TLS 1.2 while Windows 11 uses TLS 1.3.
 
-### For TCP sockets networking
+#### For SslStream
 
 <xref:System.Net.Security.SslStream>, using .NET Framework 4.7 and later versions, defaults to the OS choosing the best security protocol and version. To get the default OS best choice, if possible, don't use the method overloads of <xref:System.Net.Security.SslStream> that take an explicit <xref:System.Security.Authentication.SslProtocols> parameter. Otherwise, pass <xref:System.Security.Authentication.SslProtocols.None?displayProperty=nameWithType>. We recommend that you don't use <xref:System.Security.Authentication.SslProtocols.Default>; setting `SslProtocols.Default` forces the use of SSL 3.0 /TLS 1.0 and prevents TLS 1.2.
 
 Don't set a value for the <xref:System.Net.ServicePointManager.SecurityProtocol> property (for HTTP networking).
 
 Don't use the method overloads of <xref:System.Net.Security.SslStream> that take an explicit <xref:System.Security.Authentication.SslProtocols> parameter (for TCP sockets networking). When you retarget your app to .NET Framework 4.7 or later versions, you'll be following the best practices recommendation.
 
-The remainder of this topic is not relevant when targeting .NET Framework 4.7 or later versions for TCP sockets networking.
-
 <a name="wcf-tcp-cert"></a>
 
-### For WCF TCP transport using transport security with certificate credentials
+## For WCF Applications
+
+### If your app targets .NET Framework 4.7 or later versions
+
+The following sections show how to verify you're not using a specific TLS or SSL version.
```

Comment:
I am not sure with this sentence (and same one in the next L3 section), the content does say anything about verifying which version is being used, only how to configure the app.